File: masar_cable/custom/quotation/quotation.py
import frappe
import erpnext
import math
from frappe.utils import flt, get_datetime, getdate, date_diff, cint, nowdate, get_link_to_form, time_diff_in_hours
from frappe import throw, msgprint, _

@frappe.whitelist()
def apply_vfcu(doctype,name):
	quotation_doc = frappe.get_doc(doctype, name)
	for d in quotation_doc.get("items"):
		item_doc = frappe.get_doc("Item", d.item_code)
		d.rate = d.rate+ item_doc.vfcu *(quotation_doc.lme1-quotation_doc.lme2);
		frappe.msgprint(str(d.rate))

File: masar_cable/custom/quotation/quotation.js
frappe.ui.form.on("Quotation", "apply_vfcu", function(frm) {
show_alert("Prices are changed",5);
if(!frm.doc.current_lme) return;

frm.set_value("lme2",frm.doc.lme1);
frm.set_value("lme1",frm.doc.current_lme);
$.each(frm.doc.items, function(i, d) {
        frappe.call({
          "method": "frappe.client.get",
          args: {
            doctype: "Item",
            name: d.item_code
          },
          callback: function (data) {
            if(data.message.vfcu) {

              d.rate = d.rate+ data.message.vfcu *(frm.doc.lme1-frm.doc.lme2);
            }
          }
        });
    });
    cur_frm.save();
});
